src/libcore/iter.rs

```diff
@@ -1009,8 +1009,17 @@ pub trait Iterator {
         (ts, us)
     }
 
-    /// Creates an iterator that clones the elements it yields. Useful for
-    /// converting an Iterator<&T> to an Iterator<T>.
+    /// Creates an iterator that clones the elements it yields.
+    ///
+    /// This is useful for converting an Iterator<&T> to an Iterator<T>,
+    /// so it's a more convenient form of `map(|&x| x)`.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// let v_cloned: Vec<_> = a.iter().cloned().collect();
```

Comment:
What is `a`? And the code fence isn't closed.

(On Mobile, maybe getting a weird render)